WBooks/Views/BookIndex/BookInformation/BookInformationViewModel.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  BookInformationViewModel.swift
+//  WBooks
+//
+//  Created by Diego Quirós on 3/22/18.
+//  Copyright © 2018 Wolox. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import ReactiveSwift
+import enum Result.NoError
+
+internal final class BookInformationViewModel {
+    
+    var bookViewModel: BookViewModel
+    private var _comments = MutableProperty<[Comment]>([])
+    let comments: Property<[Comment]>
+    private var _rents = MutableProperty<[Rent]>([])
+    let rents: Property<[Rent]>
+    private let wBooksRepository: WBooksRepository = RepositoryCreator.shared.bookRepository
+    
+    init(bookViewModel: BookViewModel) {
```

Comment:
Always try to minimize dependencies, you requiring the entire `BookViewModel` just to use the id to fetch comments and rents.